test/test_wof_1_1.py

```diff
@@ -47,7 +47,7 @@ def response_to_StringIO(self, response, root_name):
         """
         Converts WOF method response to StringIO object.
         """
-        response_output = io.BytesIO()
+        response_output = io.StringIO()
```

Comment:
It seems that this test expected `StringIO`, we can revisit once we move to Python 3.